pkgs/applications/audio/goattracker/default.nix

```
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchurl
+, unzip
+, makeDesktopItem
+, imagemagick
+, SDL
+}:
+
+let desktopItem = makeDesktopItem {
+  type = "Application";
+  name = "goattracker";
+  desktopName = "GoatTracker 2";
+  genericName = "Music Tracker";
+  exec = "goattrk2";
+  icon = "goattracker";
+  categories = "Multimedia;";
+};
+
+in stdenv.mkDerivation rec {
+  pname = "goattracker";
+  version = "2.75";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/goattracker2/GoatTracker%202/${version}/GoatTracker_${version}.zip";
```

Untested, but I've seen some shorter urls like that in other packages.
```suggestion
    url = "mirror://sourceforge/goattracker2/GoatTracker_${version}.zip";
```

---

Seems to work, thanks!